packages/typespec-test/test/chatApi_modular/generated/typespec-ts/src/rest/clientDefinitions.ts

```diff
@@ -8,10 +8,10 @@ import { Client, StreamableMethod } from "@azure-rest/core-client";
 export interface CreateStreaming {
   /** Creates a new streaming chat completion. */
   post(
-    options?: CreateStreamingParameters,
+    options: CreateStreamingParameters,
```

Comment:
this is expected, with the previous tsp 
```
  createStreaming(
    ...StreamingChatCompletionOptions<TState, TContext>,
  ): ChatCompletionChunk<TState, TContext>;
```
the bodyParameter is undefined, and we define the parameter required as 
`parameters?.bodyParameter?.optional === false`

Reply:
issue tracked https://github.com/Azure/typespec-azure/issues/1149

Reply:
Create this https://github.com/Azure/autorest.typescript/issues/2677 to track our codegen work.